### PR TITLE
Refs #34758: Match rename of remote_execution::ssh to remote_executio…

### PR DESCRIPTION
**Pull request by ehelms**
…n::script

See https://github.com/theforeman/puppet-foreman_proxy/pull/734

### CODE REVIEW

File: config/foreman-proxy-content.migrations/220411132130-rename-rex-ssh-rex-script.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+if answers['foreman_proxy::plugin::remote_execution::ssh']
+  answers['foreman_proxy::plugin::remote_execution::script'] = answers['foreman_proxy::plugin::remote_execution::ssh']
+  answers.delete('foreman_proxy::plugin::remote_execution::ssh')
```
Comment (ekohl) [inline config/foreman-proxy-content.migrations/220411132130-rename-rex-ssh-rex-script.rb]:
```suggestion
  answers['foreman_proxy::plugin::remote_execution::script'] = answers.delete('foreman_proxy::plugin::remote_execution::ssh')
```

File: config/katello.migrations/220411132123-rename-rex-ssh-rex-script.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+unless answers['foreman_proxy::plugin::remote_execution::ssh'].nil?
```
Comment (ekohl) [inline config/katello.migrations/220411132123-rename-rex-ssh-rex-script.rb]: Any reason you have a different migration here than the others?

**Review by ekohl — CHANGES_REQUESTED** [review]
I'd prefer if you all have the same timestamps so it's easy to correlate. The easiest way to achieve this is to call `./bin/create-migration` with all scenarios it applies to.

**Comment by adamruzicka** [comment]
Also inconsistent dashes vs underscores in migration file names?

**Comment by ekohl** [comment]
Sadly that was already the case before and now we at least (try) to keep them consistent per scenario. In https://github.com/theforeman/foreman-installer/pull/736 I tried to align them but didn't get around to finishing it.

File: config/katello.migrations/220411132123-rename-rex-ssh-rex-script.rb (hunk 2 above)
Comment (ehelms) [reply config/katello.migrations/220411132123-rename-rex-ssh-rex-script.rb]: It's like ordering ice cream but getting two flavors, its still ice cream. j/k total copy/paste failure.

Comment (ekohl) [reply config/katello.migrations/220411132123-rename-rex-ssh-rex-script.rb]: It's why I got to the workflow of:
```
cat <<EOF | ./bin/create-migration name $scenario
here is the content
EOF
```

**Comment by ehelms** [comment]
[test foreman-installer]

**Comment by ehelms** [comment]
@evgeni This is also ready now

**Review by evgeni — APPROVED** [review]

**Pull request merged**